src/contracts/services/moneyMarket.ts

```diff
@@ -572,6 +572,83 @@ async function batchQueryMoneyMarketGetCollateral({
   }));
 }
 
+/**
+ * query the Public Logs for money market contracts
+ * NOT FOR PRODUCTION USE, CONTRACT IS IN DEVELOPMENT ON TESTNET ONLY
+ */
+/**
+ * Query the public events of a money market contract using RxJS
+ */
+function queryMoneyMarketPublicEvents$({
```

Comment:
add the query router w/block height validation. So essentially use a batch query like the other examples in this file even though this won't be a true "batch" where we are getting data from multiple contracts at once. We can do singleton queries using the query router, we have other examples of this in shadejs.  This will help us ensure we are getting up to date logs that pass the block height validation. I would like to make this the standard for ALL queries in shadejs even though some legacy ones don't have it yet. It is best practice to have block height validation available.

Reply:
Ok i added minblockheight validation if you can check this and let me know if i am doing it right.